jobs/models.py

```diff
@@ -69,7 +68,81 @@ def set_expiration_date(self):
                 self.expiration_date = self.published_date + timedelta(60)
                 self.save()
```

Comment:
Probably - can you please fix
On 12 Feb 2015 16:27, "Marysia" <notifications@github.com> wrote:

> In jobs/models.py
> <https://github.com/djangogirlsjobs/djangogirls/pull/7#discussion_r24594189>
> :
>
> > @@ -69,7 +68,81 @@ def set_expiration_date(self):
> >                  self.expiration_date = self.published_date + timedelta(60)
> >                  self.save()
>
> Maybe thet was just a gap of concentration or something
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/djangogirlsjobs/djangogirls/pull/7/files#r24594189>.
>